crypto-core/src/main/java/com/palantir/crypto2/io/CryptoStreamFactory.java

```diff
@@ -83,7 +96,7 @@ public static InputStream decrypt(InputStream input, KeyMaterial keyMaterial, St
      * cipher {@code algorithm}. When OpenSSL is available an implementation that uses AES-NI will be returned.
      */
     public static OutputStream encrypt(OutputStream output, KeyMaterial keyMaterial, String algorithm) {
-        return encrypt(output, keyMaterial, algorithm, false);
+        return encrypt(output, keyMaterial, algorithm, !OPENSSL_IS_AVAILABLE.get());
```

Comment:
I think things may read a bit better if we rewrite the encrypt method below like:
```
static OutputStream encrypt(OutputStream output, KeyMaterial keyMaterial, String algorithm, boolean forceJce) {
    if (algorithm.equals(AES_ALGORITHM) && OPENSSL_IS_AVAILABLE.get() && !forceJce) {
        return createApacheEncryptedStream(output, keyMaterial);
    } else {
        return createDefaultEncryptedStream(output, keyMaterial, algorithm);
    }
```

I also don't remember which tests needed the `forceJce` flag so may be able to consider removing that completely now as well.

Reply:
Removal would make it impossible to test both paths on the same CI run, so I don't think we should do that unless we're removing one of the two paths entirely :-)
Let's hold off cleaning up this codebase for an isolated commit rather than coupling it with logging cleanup, I'll add a minimal refactor to retain the `false` arg for the time being.